src/checkout/checkout-service.ts

```diff
@@ -520,6 +522,20 @@ export default class CheckoutService {
         return this._dispatch(action, { queueId: 'customerStrategy' });
     }
 
+    /**
+     * Sends a email that contains a single-use sign-in link. When clicked, this link
+     * signs in the customer without requiring any password.
+     *
+     * @param email - The email to be sent the sign-in link.
+     * @param options - Options for the send email request.
+     * @returns A promise that resolves to the current state.
+     */
+    sendSignInEmail(email: string, options?: RequestOptions): Promise<CheckoutSelectors> {
```

Comment:
Perhaps we can put `@internal` annotation so these new methods don't get published to our docs until this feature is ready to go live?